spec/controllers/sessions_controller_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'spec_helper'
+
+describe SessionsController do
+
+  describe "GET new" do
+    it "redirects to the home path if the user is logged in" do
+      session[:user_id] = Fabricate(:user).id
+      get :new
+      expect(response).to redirect_to home_path
+    end
+
+    it "renders the new template for unauthenticated users" do
+      get :new
+      expect(response).to render_template :new
+    end
+  end
+
+  describe "POST create" do
+    it "sets the user variable based on the email entered" do
+      current_user = Fabricate(:user)
+      post :create, email: User.first.email
+      expect(assigns(:user)).to eq(current_user)
+    end
+
+    it "sets session[:user_id] to the @user.id when the @user is authenticated" do
+      user = Fabricate(:user)
+      post :create, email: user.email, password: user.password
+      expect(session[:user_id]).to eq(user.id)
+      expect(response).to redirect_to home_path
```

Comment:
This expectation doesn't belong here, it doesn't have anything to do with the description of this spec. You should move it to its own spec.